.github/workflows/build.yml

```diff
@@ -26,24 +26,30 @@ jobs:
       matrix:
         rust:
           - stable
-          - beta
+          # XXX TBD SKIP FOR NOW TO SPEED UP CI BUILD TESTING
+          # - beta
           - nightly
         os:
-          - ubuntu-latest
-          - windows-latest
+          # XXX TBD SKIP FOR NOW TO SPEED UP CI BUILD TESTING
+          # - ubuntu-latest
+          # - windows-latest
           - macos-latest
```

Comment:
CI testing seems to start progressing faster with Ubuntu & Windows out of the matrix.

My idea is to remove Windows from here & add a single job that *only* does build & test with all features on Windows.

I may consider a similar idea for Ubuntu in this fork.